docs/deployment/docker.md

```diff
@@ -213,7 +214,11 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     apt-get install -y --no-install-recommends libsqlite3-dev && \
     rm -rf /var/lib/apt/lists/*
 
+RUN corepack enable && npm install -g corepack@0.27.0 && corepack install --global yarn@1.22.22
```

Comment:
We've done some testing with `corepack` and had issues with it, we'd like to hold off until Node 20 is LTS in November. If you'd like to help us add support then that would be awesome!